src/test/java/org/opensearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -322,9 +323,12 @@ private void verifyPatch(final boolean sendAdminCert, Header... restAdminHeader)
                 HttpStatus.SC_CREATED);
 
         // Add enabled service account then get it
+
+        rh.sendAdminCertificate = sendAdminCert;
```

Comment:
Don't think we need this, as we are not changing `sendAdminCert` value at any point below

Reply:
Oops nice catch.